azure/services/managedclusters/spec.go

```diff
@@ -380,429 +389,209 @@ func (s *ManagedClusterSpec) Parameters(ctx context.Context, existing interface{
 			// https://golang.org/src/net/ip.go#L48
 			ip[15] = byte(10)
 			dnsIP := ip.String()
-			managedCluster.Properties.NetworkProfile.DNSServiceIP = &dnsIP
-		} else {
-			managedCluster.Properties.NetworkProfile.DNSServiceIP = s.DNSServiceIP
```

Comment:
Do we no longer want to set the `DNSServiceIP` to the existing one? I'm assuming ASO will handle it but just making sure

Reply:
It's set above. This is a not-entirely-shameless code coverage hack.